CMakeLists.txt

```diff
@@ -418,7 +420,12 @@ else()
 endif()
 set(MIOPEN_SYSTEM_FIND_DB_SUFFIX "${MIOPEN_BACKEND}" CACHE PATH "Filename suffix for the system find-db files")
 
-set(MIOPEN_PACKAGE_REQS "hip-rocclr")
+# PR-2391 Add the ability to log function calls to roctx.
+# This allows attached profilers to see which MIOpen calls are being called by application and which kernels are being invoked by MIOpen.
+# Enabled via the MIOPEN_ENABLE_LOGGING_ROCTX env var.
+if(NOT WIN32)
+    set(MIOPEN_PACKAGE_REQS "${MIOPEN_PACKAGE_REQS}, roctracer")
```

Comment:
[Question] @junliume This makes roctracer and roctx64 mandatory. Is it possible that these libraries are not available on the target system (for example due to outdated ROCm, which we would still like to support)?

Reply:
@atamazov thanks and I have the same concern. Dependency on `dev` tools should be "optional" IMHO. However, https://github.com/ROCmSoftwarePlatform/MIOpen/issues/2502 originated from a UIF docker issue where only during runtime the missing dependency is reported. i.e. I think we should not build two sets of packages, one depends on rocTracer while the other does not.
Meanwhile, I think we can and should put a version check here, like
```
if(NOT WIN32 && MIOPEN_VERSION >= 3.0) 
```

Reply:
@junliume We can add CMake flag and pass it into our sources via config.txt (like MIOPEN_USE_COMGR). We can use it as a workaround if libs are missing.